includes/validation/class-amp-invalid-url-post-type.php

```diff
@@ -583,92 +626,78 @@ public static function output_custom_column( $column_name, $post_id ) {
+			case AMP_Validation_Error_Taxonomy::FOUND_ELEMENTS_AND_ATTRIBUTES:
+				$items = array();
 				if ( ! empty( $error_summary[ AMP_Validation_Error_Taxonomy::REMOVED_ELEMENTS ] ) ) {
-					$items = array();
 					foreach ( $error_summary[ AMP_Validation_Error_Taxonomy::REMOVED_ELEMENTS ] as $name => $count ) {
 						if ( 1 === intval( $count ) ) {
 							$items[] = sprintf( '<code>%s</code>', esc_html( $name ) );
 						} else {
 							$items[] = sprintf( '<code>%s</code> (%d)', esc_html( $name ), $count );
 						}
 					}
-					echo implode( ', ', $items ); // WPCS: XSS OK.
-				} else {
-					esc_html_e( '--', 'amp' );
 				}
-				break;
-			case AMP_Validation_Error_Taxonomy::REMOVED_ATTRIBUTES:
 				if ( ! empty( $error_summary[ AMP_Validation_Error_Taxonomy::REMOVED_ATTRIBUTES ] ) ) {
-					$items = array();
 					foreach ( $error_summary[ AMP_Validation_Error_Taxonomy::REMOVED_ATTRIBUTES ] as $name => $count ) {
 						if ( 1 === intval( $count ) ) {
-							$items[] = sprintf( '<code>%s</code>', esc_html( $name ) );
+							$items[] = sprintf( '<code>[%s]</code>', esc_html( $name ) );
 						} else {
-							$items[] = sprintf( '<code>%s</code> (%d)', esc_html( $name ), $count );
+							$items[] = sprintf( '<code>[%s]</code> (%d)', esc_html( $name ), $count );
 						}
 					}
-					echo implode( ', ', $items ); // WPCS: XSS OK.
+				}
+				if ( ! empty( $items ) ) {
+					echo implode( ',<br/>', $items ); // WPCS: XSS OK.
 				} else {
 					esc_html_e( '--', 'amp' );
 				}
 				break;
 			case AMP_Validation_Error_Taxonomy::SOURCES_INVALID_OUTPUT:
 				if ( isset( $error_summary[ AMP_Validation_Error_Taxonomy::SOURCES_INVALID_OUTPUT ] ) ) {
-					$sources = array();
-					foreach ( $error_summary[ AMP_Validation_Error_Taxonomy::SOURCES_INVALID_OUTPUT ] as $type => $names ) {
-						foreach ( array_unique( $names ) as $name ) {
-							$sources[] = sprintf( '%s: <code>%s</code>', esc_html( $type ), esc_html( $name ) );
+					$sources = $error_summary[ AMP_Validation_Error_Taxonomy::SOURCES_INVALID_OUTPUT ];
+					$output  = array();
+
+					if ( isset( $sources['plugin'] ) ) {
+						$output[]                  = '<div class="source">';
+						$count                     = count( array_unique( $sources['plugin'] ) );
+						$sources_container_classes = 'sources-container sources-plugins';
+						if ( 1 === $count ) {
+							$output[] = sprintf( '<span class="dashicons dashicons-admin-plugins"></span></span><strong>%s</strong><br/>', esc_html__( 'Plugin', 'amp' ) );
+						} else {
+							$output[]                   = sprintf( '<span class="dashicons dashicons-admin-plugins"></span><strong>%s (%d)</strong><span class="dashicons toggle-sources dashicons-arrow-down"></span>', esc_html__( 'Plugins', 'amp' ), $count );
+							$sources_container_classes .= ' collapsed';
 						}
+						$output[] = '<div class="' . $sources_container_classes . '">';
+						$output[] = implode( '<br/>', array_unique( $sources['plugin'] ) );
```

Comment:
This should obtain the actual names of the plugins using `get_plugin_data()` and getting the `Name` returned. If the plugin doesn't exist then show the slug (and this result is obviously stale).